statsmodels/discrete/tests/test_count_model.py

```diff
@@ -530,30 +530,30 @@ def test_predict_generic_zi(self):
 
 
 class TestZeroInflatedNegativeBinomialP_predict2(object):
-        @classmethod
-        def setup_class(cls):
-            data = sm.datasets.randhie.load(as_pandas=False)
-
-            cls.endog = data.endog
-            exog = data.exog
-            start_params = np.array(
-                [-2.83983767, -2.31595924, -3.9263248 , -4.01816431, -5.52251843,
-                -2.4351714 , -4.61636366, -4.17959785, -0.12960256, -0.05653484,
-                -0.21206673,  0.08782572, -0.02991995,  0.22901208,  0.0620983 ,
-                 0.06809681,  0.0841814 ,  0.185506  ,  1.36527888])
-            mod = sm.ZeroInflatedNegativeBinomialP(
-                cls.endog, exog, exog_infl=exog, p=2)
-            res = mod.fit(start_params=start_params, method="bfgs",
-                          maxiter=1000, disp=0)
-
-            cls.res = res
-
-        def test_mean(self):
-            assert_allclose(self.res.predict().mean(), self.endog.mean(),
-                            atol=0.02)
-
-        def test_zero_nonzero_mean(self):
-            mean1 = self.endog.mean()
-            mean2 = ((1 - self.res.predict(which='prob-zero').mean()) *
-                     self.res.predict(which='mean-nonzero').mean())
-            assert_allclose(mean1, mean2, atol=0.2)
+    @classmethod
+    def setup_class(cls):
+        data = sm.datasets.randhie.load(as_pandas=False)
+
+        cls.endog = data.endog
+        exog = data.exog
+        start_params = np.array([
+            -2.83983767, -2.31595924, -3.9263248,  -4.01816431, -5.52251843,
+            -2.4351714,  -4.61636366, -4.17959785, -0.12960256, -0.05653484,
```

Comment:
fill with trailing zeros instead of destroying the alignment

Reply:
alignment of the numbers is not affected since this substitutes ", " for " ,"

Reply:
ok, it works here, I didn't realize this part, but the alignment changed in other PRs, and below, last line of numbers, where there is no negative sign.